File: csharp/SayWhatStarterWebhook/Models/Messages/GoogleSimpleResponsesMessage.cs
using System.Collections.Generic;

namespace SayWhatStarterWebhook.Models.Messages
{
    public class GoogleSimpleResponsesMessage : GooglePlatformMessage
    {
        public GoogleSimpleResponses SimpleResponses { get; set; }

        public static GoogleSimpleResponsesMessage Create(GoogleSimpleResponse response)
        {
            return new GoogleSimpleResponsesMessage
            {
                SimpleResponses = new GoogleSimpleResponses
                {
                    SimpleResponses = new List<GoogleSimpleResponse> {response}
                }
            };
        }
        public static GoogleSimpleResponsesMessage Create(string displayText, string textToSpeech)
        {
            return Create(new GoogleSimpleResponse
            {
                DisplayText = displayText,
                TextToSpeech = textToSpeech
            });
        }
    }
}

File: jvm/src/main/kotlin/org/faziodev/saywhatstarter/types/messages/GoogleSimpleResponsesMessage.kt
package org.faziodev.saywhatstarter.types.messages

class GoogleSimpleResponsesMessage(val simpleResponses: GoogleSimpleResponses) : GooglePlatformMessage() {
    companion object {
        fun create(responses: List<GoogleSimpleResponse>): GoogleSimpleResponsesMessage {
            return GoogleSimpleResponsesMessage(
                GoogleSimpleResponses(
                    responses
                )
            )
        }
        fun create(response: GoogleSimpleResponse): GoogleSimpleResponsesMessage = this.create(listOf(response))
    }
}

